Common/Product/SharedProject/ProcessOutput.cs

```diff
@@ -237,27 +234,18 @@ sealed class ProcessOutput : IDisposable {
         ) {
             var outFile = Path.GetTempFileName();
             var errFile = Path.GetTempFileName();
-            var psi = new ProcessStartInfo("cmd.exe");
-            psi.CreateNoWindow = true;
-            psi.WindowStyle = ProcessWindowStyle.Hidden;
-            psi.UseShellExecute = true;
-            psi.Verb = "runas";
-
-            string args;
-            if (quoteArgs) {
-                args = string.Join(" ", arguments.Where(a => a != null).Select(QuoteSingleArgument));
-            } else {
-                args = string.Join(" ", arguments.Where(a => a != null));
-            }
-            psi.Arguments = string.Format("/S /C \"{0} {1} >>{2} 2>>{3}\"",
-                QuoteSingleArgument(filename),
-                args,
-                QuoteSingleArgument(outFile),
-                QuoteSingleArgument(errFile)
-            );
-            psi.WorkingDirectory = workingDirectory;
```

Comment:
Hrm... I'm concerned about the implications of not setting the WorkingDirectory property. In particular, the working directory property has different implications depending on the value of `UseShellExecute`. Did you verify the edge cases here? 
https://msdn.microsoft.com/en-us/library/system.diagnostics.processstartinfo.workingdirectory(v=vs.110).aspx